### PR TITLE
Fixed #30934 -- Included database alias in django.db.backends log messages.

### PR DESCRIPTION
**Pull request by pope1ni**
ticket-30934.

Resurrects #11994 and finishes it off - it was ever so close!

### CODE REVIEW

File: docs/ref/logging.txt [hunk 1]
```diff
@@ -188,6 +189,10 @@ This logging does not include framework-level initialization (e.g.
 ``COMMIT``, and ``ROLLBACK``). Turn on query logging in your database if you
 wish to view all database queries.
 
+.. versionchanged:: 4.0
+
+    Messages now include the ``alias`` where the query was executed.
```
Comment (felixxm) [inline docs/ref/logging.txt:194]: Maybe:
```suggestion
    The database ``alias`` was added to log messages.
```

File: docs/releases/4.0.txt [hunk 2]
```diff
@@ -226,7 +226,8 @@ Internationalization
 Logging
 ~~~~~~~
 
-* ...
+* The alias of the database used in a SQL call is now passed as extra context
```
Comment (felixxm) [inline docs/releases/4.0.txt:229]:
```suggestion
* The alias of the database used in an SQL call is now passed as extra context
```

**Review by felixxm — COMMENTED** [review]
@pope1ni Thanks for bringing it to the finish line :star: Two nitpicks :detective:

File: docs/releases/4.0.txt (hunk 2 above)
Comment (pope1ni) [reply docs/releases/4.0.txt:229]: Ok. I went with what you have based on that being using in the [Wikipedia article](https://en.wikipedia.org/wiki/SQL).

It's actually complicated based on how you pronounce it though:

1. "S-Q-L" → ...an SQL...
2. "Sequel" → ...a SQL...

Fun.

Comment (felixxm) [reply docs/releases/4.0.txt:229]: I know, but in Django's docs we prefer `an`.

Comment (pope1ni) [reply docs/releases/4.0.txt]: Now I know!

**Pull request merged**